### PR TITLE
feat-cart

### PR DESCRIPTION
**Pull request by iamsaurav28**
Screenshot (97)

### CODE REVIEW

File: comp-layout/cart.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+</head>
+<body>
+
+     <header class="home-header">
+     <div class="home-container">
+          <div class="navbar">
+               <div class="symbol">
+                    <div class="symbol-name">SAADGI UI</div>
+               </div>
+               <img  class="search-icon" src="https://img.icons8.com/material-outlined/24/000000/google-web-search.png"/> <input  class="search-bar" type="text" placeholder="search here" />
```
Comment (krishanu2911) [inline comp-layout/cart.html:22]: add the alt tag

File: comp-layout/cart.html [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+          </tr>
+
+          <tr>
+              <td>
+                   <div class="product-detail">
+                        <img src="/comp-images/tv.jpg" />
+                        <div>
+                             <p>MI SMART TV BRAND</p>
+                             <small>Price: $354.00</small>
+                             <a href="">Remove</a>
```
Comment (krishanu2911) [inline comp-layout/cart.html]: do not leave href empty

File: comp-layout/cart.html [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+     <table>
+          <tr>
+               <th>Product</th>
+               <th>Quantity</th>
+               <th>Subtotal</th>
+          </tr>
+          <tr>
+               <td>
+                    <div class="product-detail">
+                         <img src="/comp-images/headphones - Copy.jpg" />
```
Comment (krishanu2911) [inline comp-layout/cart.html]: add the alt tag

**Review by krishanu2911 — COMMENTED** [review]
Everything is good to go added few correction

File: comp-layout/cart.html (hunk 1 above)
Comment (iamsaurav28) [reply comp-layout/cart.html:22]: okay done

File: comp-layout/cart.html (hunk 2 above)
Comment (iamsaurav28) [reply comp-layout/cart.html]: okay done

File: comp-layout/cart.html (hunk 3 above)
Comment (iamsaurav28) [reply comp-layout/cart.html]: okkk done

File: comp-layout/cart.html [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+<head>
+     <meta charset="UTF-8">
+     <meta http-equiv="X-UA-Compatible" content="IE=edge">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,600;0,700;1,300;1,500&family=Roboto+Condensed&family=Russo+One&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">     
+<link rel="stylesheet" href="/stylesheet/main.css">
+
+     <title>Product-page</title>
```
Comment (rahulcs754) [inline comp-layout/cart.html]: I think this is a cart page, not a product page. you should have change this

File: stylesheet/components/cart.css [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+     font-weight: normal;
+}
+
+td{
+     padding: 10px 5px;
+}
+
+td input{
+     width: 40px;
+     height: 30px;
```
Comment (rahulcs754) [inline stylesheet/components/cart.css:28]: try to use rem unit

File: stylesheet/components/cart.css [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+table{
+     width: 100%;
+     border-collapse: collapse;
+}
+
+.product-detail{
+     display: flex;
+     flex-wrap: wrap;
```
Comment (rahulcs754) [inline stylesheet/components/cart.css:12]: I think you should create this utility class. That will help full for you in most of component later .

**Review by rahulcs754 — APPROVED** [review]
I looked at your code that is fine. you can create some utility classes for flex center, margin auto bcoz these class will helpful for many places in the feature.

File: comp-layout/cart.html (hunk 4 above)
Comment (iamsaurav28) [reply comp-layout/cart.html]: oh yes done i have changed

File: stylesheet/components/cart.css (hunk 5 above)
Comment (iamsaurav28) [reply stylesheet/components/cart.css:28]: here i have used px everywhere, next time i ll try to keep in rem.

File: stylesheet/components/cart.css (hunk 6 above)
Comment (iamsaurav28) [reply stylesheet/components/cart.css:12]: okay i ll do it

**Review by krishanu2911 — APPROVED** [review]

**Pull request merged**